python/llm/example/GPU/HF-Transformers-AutoModels/Model/minicpm/README.md

```diff
@@ -0,0 +1,127 @@
+# minicpm
+In this directory, you will find examples on how you could apply IPEX-LLM INT4 optimizations on minicpm models on [Intel GPUs](../../../README.md). For illustration purposes, we utilize the [openbmb/MiniCPM-2B-sft-bf16](https://huggingface.co/openbmb/MiniCPM-2B-sft-bf16) as a reference minicpm model.
+
+## 0. Requirements
+To run these examples with IPEX-LLM on Intel GPUs, we have some recommended requirements for your machine, please refer to [here](../../../README.md#requirements) for more information.
+
+## Example: Predict Tokens using `generate()` API
+In the example [generate.py](./generate.py), we show a basic use case for a minicpm model to predict the next N tokens using `generate()` API, with IPEX-LLM INT4 optimizations on Intel GPUs.
+### 1. Install
+#### 1.1 Installation on Linux
+We suggest using conda to manage environment:
+```bash
+conda create -n llm python=3.11
+conda activate llm
+# below command will install intel_extension_for_pytorch==2.1.10+xpu as default
+pip install --pre --upgrade ipex-llm[xpu] --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/
+
+pip install transformers==4.37.0
```

Comment:
Is `transformers 4.37.0` necessary here? If not, let's just use the default transformers version, i.e. no need to explicitly install `transformers` here.

Reply:
Same for other places.